### PR TITLE
And2 schematics

### PR DESCRIPTION
**Pull request by MitchTansey**

### CODE REVIEW

**Comment by sifferman** [comment]
Observe this error message:

> Please confirm your contribution by adding 'MitchTansey' to docs/CONTRIBUTORS in sorted order.
   If you do not wish to go by the name 'MitchTansey', update 'name=' in your ~/.gitconfig file.
   Also ensure your name on https://github.com/ under Settings->Profile->Name
   matches your ~/.gitconfig 'name=' field.

**Comment by sifferman** [comment]
Just took a look at the cells. Thanks for your work!

Your 0 and 1 cells looks good.

Your 1 and 2 cells have a couple problems:

* There should not be any diagonal lines
* The inverters should all be facing the same direction.
* The inverters should probably all be stacked vertically. I don't see a reason to have them side by side, since there are a max of 4.

Feel free to take a look at the new nor2 cells for inspiration: <https://github.com/sifferman/sky130_schematics/releases/latest>

**Comment by MitchTansey** [comment]
Ok, I implemented those changes, let me know if anything else needs reworking, thanks!

**Comment by sifferman** [comment]
> Observe this error message:
> 
> > Please confirm your contribution by adding 'MitchTansey' to docs/CONTRIBUTORS in sorted order.
> > If you do not wish to go by the name 'MitchTansey', update 'name=' in your ~/.gitconfig file.
> > Also ensure your name on https://github.com/ under Settings->Profile->Name
> > matches your ~/.gitconfig 'name=' field.

Please implement this fix. Your name needs to be corrected in your .gitconfig and GitHub account. Then, you need to add your name to docs/CONTRIBUTORS

File: schematics/and2/sky130_fd_sc_hd__and2_0.sch [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+N 200 60 200 80 {lab=VGND}
+N 160 -50 160 30 {lab=#net1}
+N 200 -20 300 -20 {lab=X}
+C {ipin.sym} -270 -100 0 0 {name=p1 lab=A}
+C {opin.sym} -200 -100 0 0 {name=p2 lab=X}
+C {ipin.sym} -270 -80 0 0 {name=p18 lab=B}
+C {ipin.sym} -270 -60 0 0 {name=p19 lab=VGND}
+C {ipin.sym} -270 -40 0 0 {name=p20 lab=VNB}
+C {ipin.sym} -270 -20 0 0 {name=p21 lab=VPB}
+C {ipin.sym} -270 0 0 0 {name=p22 lab=VPWR}
```
Comment (sifferman) [inline schematics/and2/sky130_fd_sc_hd__and2_0.sch:24]:
```suggestion
C {ipin.sym} -270 -100 0 0 {name=p1 lab=A}
C {ipin.sym} -270 -80 0 0 {name=p18 lab=B}
C {ipin.sym} -270 -60 0 0 {name=p19 lab=VGND}
C {ipin.sym} -270 -40 0 0 {name=p20 lab=VNB}
C {ipin.sym} -270 -20 0 0 {name=p21 lab=VPB}
C {ipin.sym} -270 0 0 0 {name=p22 lab=VPWR}
C {opin.sym} -200 -100 0 0 {name=p2 lab=X}
```

File: schematics/and2/sky130_fd_sc_hd__and2_1.sch [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+N 200 60 200 80 {lab=VGND}
+N 160 -50 160 30 {lab=#net1}
+N 200 -20 300 -20 {lab=X}
+C {ipin.sym} -270 -100 0 0 {name=p1 lab=A}
+C {opin.sym} -200 -100 0 0 {name=p2 lab=X}
+C {ipin.sym} -270 -80 0 0 {name=p18 lab=B}
+C {ipin.sym} -270 -60 0 0 {name=p19 lab=VGND}
+C {ipin.sym} -270 -40 0 0 {name=p20 lab=VNB}
+C {ipin.sym} -270 -20 0 0 {name=p21 lab=VPB}
+C {ipin.sym} -270 0 0 0 {name=p22 lab=VPWR}
```
Comment (sifferman) [inline schematics/and2/sky130_fd_sc_hd__and2_1.sch:24]:
```suggestion
C {ipin.sym} -270 -100 0 0 {name=p1 lab=A}
C {ipin.sym} -270 -80 0 0 {name=p18 lab=B}
C {ipin.sym} -270 -60 0 0 {name=p19 lab=VGND}
C {ipin.sym} -270 -40 0 0 {name=p20 lab=VNB}
C {ipin.sym} -270 -20 0 0 {name=p21 lab=VPB}
C {ipin.sym} -270 0 0 0 {name=p22 lab=VPWR}
C {opin.sym} -200 -100 0 0 {name=p2 lab=X}
```

File: schematics/and2/sky130_fd_sc_hd__and2_2.sch [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+N 200 100 200 110 {lab=#net1}
+N 200 110 230 110 {lab=#net1}
+N 270 120 370 120 {lab=#net2}
+C {ipin.sym} -270 -100 0 0 {name=p1 lab=A}
+C {opin.sym} -200 -100 0 0 {name=p2 lab=X}
+C {ipin.sym} -270 -80 0 0 {name=p18 lab=B}
+C {ipin.sym} -270 -60 0 0 {name=p19 lab=VGND}
+C {ipin.sym} -270 -40 0 0 {name=p20 lab=VNB}
+C {ipin.sym} -270 -20 0 0 {name=p21 lab=VPB}
+C {ipin.sym} -270 0 0 0 {name=p22 lab=VPWR}
```
Comment (sifferman) [inline schematics/and2/sky130_fd_sc_hd__and2_2.sch:35]:
```suggestion
C {ipin.sym} -270 -100 0 0 {name=p1 lab=A}
C {ipin.sym} -270 -80 0 0 {name=p18 lab=B}
C {ipin.sym} -270 -60 0 0 {name=p19 lab=VGND}
C {ipin.sym} -270 -40 0 0 {name=p20 lab=VNB}
C {ipin.sym} -270 -20 0 0 {name=p21 lab=VPB}
C {ipin.sym} -270 0 0 0 {name=p22 lab=VPWR}
C {opin.sym} -200 -100 0 0 {name=p2 lab=X}
```

File: schematics/and2/sky130_fd_sc_hd__and2_4.sch [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+N 140 -220 170 -220 {}
+N 140 0 140 220 {}
+N 140 220 170 220 {}
+C {ipin.sym} -270 -100 0 0 {name=p1 lab=A}
+C {opin.sym} -200 -100 0 0 {name=p2 lab=X}
+C {ipin.sym} -270 -80 0 0 {name=p18 lab=B}
+C {ipin.sym} -270 -60 0 0 {name=p19 lab=VGND}
+C {ipin.sym} -270 -40 0 0 {name=p20 lab=VNB}
+C {ipin.sym} -270 -20 0 0 {name=p21 lab=VPB}
+C {ipin.sym} -270 0 0 0 {name=p22 lab=VPWR}
```
Comment (sifferman) [inline schematics/and2/sky130_fd_sc_hd__and2_4.sch:52]:
```suggestion
C {ipin.sym} -270 -100 0 0 {name=p1 lab=A}
C {ipin.sym} -270 -80 0 0 {name=p18 lab=B}
C {ipin.sym} -270 -60 0 0 {name=p19 lab=VGND}
C {ipin.sym} -270 -40 0 0 {name=p20 lab=VNB}
C {ipin.sym} -270 -20 0 0 {name=p21 lab=VPB}
C {ipin.sym} -270 0 0 0 {name=p22 lab=VPWR}
C {opin.sym} -200 -100 0 0 {name=p2 lab=X}
```

**Review by sifferman — CHANGES_REQUESTED** [review]
Also, if you pull from main, note this new error message your schematics get. You need to reorder the pins in your .sch files

> ERROR: .subckt definitions do not match between "netlists/and2/sky130_fd_sc_hd__and2_1.spice" and "references/and2/sky130_fd_sc_hd__and2_1.spice"
>  Expected: .subckt sky130_fd_sc_hd__and2_1 A B VGND VNB VPB VPWR X
>  Received: .subckt sky130_fd_sc_hd__and2_1 A X B VGND VNB VPB VPWR
>  Please reorder the ports inside "schematics/and2/sky130_fd_sc_hd__and2_1.sch"

**Comment by MitchTansey** [comment]
ok i added myself to the contributors and everything, i also committed all your suggested changes. for some reason, it says that changes are still requested which I'm not sure about

**Comment by sifferman** [comment]
I'll recommend changing your name to "Mitch Tansey". Please go through the process to do that

**Review by sifferman — APPROVED** [review]

**Comment by MitchTansey** [comment]
well im confused because my .gitconfig has to match my username on github, but sean put they're name in the CONTRIBUTORS when their github username is not sean li

**Comment by sifferman** [comment]
Yes, this is the contents of my `"~/.gitconfig"`
```gitconfig
[user]
	name = Ethan Sifferman
	email = ethan@sifferman.dev
```

**Comment by MitchTansey** [comment]
Ok I changed it

**Comment by sifferman** [comment]
You should also change your name on GitHub

**Comment by MitchTansey** [comment]
I see ok I changed my name on github to "Mitchell Tansey"

**Comment by MitchTansey** [comment]
Is there a way to run the checks again without committing anything? Because nothing needs to be changed in my actual repo, it was just a github thing

**Comment by sifferman** [comment]
You're all set! Thanks for your work!

Note that I squashed this PR. So for your next branch, you will need to reset your git branch back to my main branch. Otherwise, you will get weird errors

**Pull request merged**